src/sync/streaming/SSEHandler/index.ts

```diff
@@ -82,11 +82,16 @@ export default function SSEHandlerFactory(log: ILogger, pushEmitter: IPushEventE
             parsedData.changeNumber,
             parsedData.segmentName);
           break;
+        // @TODO remove
         case MY_SEGMENTS_UPDATE:
           pushEmitter.emit(MY_SEGMENTS_UPDATE,
             parsedData,
             channel);
           break;
+        case MY_SEGMENTS_UPDATE_V2:
+          pushEmitter.emit(MY_SEGMENTS_UPDATE_V2,
+            parsedData);
```

Comment:
When polishing, consider standardizing this file so all events are emitted with the type and parsedData (like this one), and inside the handler you pick up what you want from there. Instead of some events sending the full data and some others picking up three properties that end up being sent. (like for the kill)

Reply:
With that change it could potentially be refactored for all cases that emit an event: 

case SPLIT_UPDATE
case MY_SEGMENTS_UPDATE
...
case SPLIT_KILL: 
  pushEmitter.emit(parsedData.type, parsedData);
  break;
case occupancy...